app/src/main/res/values/strings.xml

```diff
@@ -245,41 +245,41 @@
-    <string name="warning_no_encryption_key">Are you sure you do not want to use any encryption key?</string>
-    <string name="warning_permanently_delete_nodes">Are you sure you want to permanently delete the selected nodes?</string>
+    <string name="warning_no_encryption_key">Continue with unencrypted database?</string>
+    <string name="warning_permanently_delete_nodes">Permanently delete selected nodes?</string>
     <string name="version_label">Version %1$s</string>
     <string name="build_label">Build %1$s</string>
-    <string name="configure_biometric">Biometric prompt is supported but not set up.</string>
-    <string name="keystore_not_accessible">The keystore is not properly initialized.</string>
-    <string name="open_biometric_prompt_unlock_database">Open the biometric prompt to unlock the database</string>
-    <string name="open_biometric_prompt_store_credential">Open the biometric prompt to store credentials</string>
+    <string name="configure_biometric">Biometric input is supported but not set up.</string>
+    <string name="keystore_not_accessible">Could not read keys.</string>
+    <string name="open_biometric_prompt_unlock_database">Open the biometric input to unlock the database</string>
+    <string name="open_biometric_prompt_store_credential">Open the biometric input to store credentials</string>
     <string name="biometric_prompt_store_credential_title">Save biometric recognition</string>
     <string name="biometric_prompt_store_credential_message">Warning: You still need to remember your master password if you use biometric recognition.</string>
-    <string name="biometric_prompt_extract_credential_title">Open database with biometric recognition</string>
-    <string name="biometric_prompt_extract_credential_message">Extract database credential with biometric data</string>
+    <string name="biometric_prompt_extract_credential_title">Database unlocking with biometry</string>
+    <string name="biometric_prompt_extract_credential_message">Fetch database credential using biometric data recognition</string>
     <string name="encrypted_value_stored">Encrypted password stored</string>
-    <string name="biometric_invalid_key">Can not read the biometric key. Please delete it and repeat the biometric recognition procedure.</string>
+    <string name="biometric_invalid_key">Retry biometric recognition</string>
     <string name="biometric_not_recognized">Could not recognize biometric</string>
     <!--This problem could be with scanning, or something else.-->
     <string name="biometric_scanning_error">Biometric error: %1$s</string>
     <string name="no_credentials_stored">This database does not have stored credential yet.</string>
-    <string name="credential_before_click_biometric_button">Type the password before clicking the biometric button.</string>
+    <string name="credential_before_click_biometric_button">Type in the password first.</string>
```

Comment:
First, but after?

Reply:
@J-Jamet I understood this to be an error thrown after the biometric button was already clicked(?)